apps/arweave/src/ar_nonce_limiter_server_worker.erl

```diff
@@ -56,12 +56,18 @@ handle_info({event, nonce_limiter, _Event}, #state{ peer = undefined } = State)
 handle_info({event, nonce_limiter, {computed_output, Args}}, State) ->
 	#state{ peer = Peer, pause_until = Timestamp } = State,
 	{SessionKey, Session, PrevSessionKey, PrevSession, Output, PartitionUpperBound} = Args,
+	CurrentStepNumber = ar_nonce_limiter:get_current_step_number(),
 	case os:system_time(second) < Timestamp of
 		true ->
 			{noreply, State};
 		false ->
-			{noreply, push_update(SessionKey, Session, PrevSessionKey, PrevSession, Output,
-					PartitionUpperBound, Peer, State)}
+			case Session#vdf_session.step_number < CurrentStepNumber of
```

Comment:
Skip over any VDF updates that have occurred while the `server_worker` was waiting on the `vdf_client` to process that last update.